locksmithctl/locksmithctl.go

```diff
@@ -82,20 +96,25 @@ type Command struct {
 }
 
 func getAllFlags() (flags []*flag.Flag) {
-	return getFlags(globalFlagset)
+	return getFlags(globalFlagSet)
 }
 
-func getFlags(flagset *flag.FlagSet) (flags []*flag.Flag) {
+func getFlags(flagSet *flag.FlagSet) (flags []*flag.Flag) {
 	flags = make([]*flag.Flag, 0)
-	flagset.VisitAll(func(f *flag.Flag) {
+	flagSet.VisitAll(func(f *flag.Flag) {
 		flags = append(flags, f)
 	})
 	return
 }
 
 func main() {
-	globalFlagset.Parse(os.Args[1:])
-	var args = globalFlagset.Args()
+	globalFlagSet.Parse(os.Args[1:])
+	globalFlagSet.VisitAll(func(f *flag.Flag) {
```

Comment:
This seems awkward; how about just setting this to the desired value in the `globalFlags` initialisation instead?